toolbox-reference_5b7bdc1d0428631d7a8a2fcc.md

```diff
@@ -215,9 +215,14 @@ Examples:
     cache store gems-$SEMAPHORE_GIT_BRANCH-revision-$(checksum Gemfile.lock) vendor/bundle
 
 Archives file or directory specified by path and associates it with key.
-Any further changes of path after the store operation completes will not
+As `cache store` uses `tar` leading `/` is automatically removed from the path.
```

Comment:
I would rephrase it as:

As `cache store` uses the `tar` utility, it automatically removes any leading `/` (if they exist) from the given `path` value.